modules/ROOT/pages/configure-hardening.adoc

```diff
@@ -0,0 +1,23 @@
+= Configure Environment Hardening on Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+By design, Runtime Fabric automatically provides multiple aspects of environment hardening for your Mule installation. 
+For example, by leveraging Docker and Kubernetes, app isolation is achieved by default.  
+
+The following hardening checklist provides additional information about environment hardening already provided by 
+Runtime Fabric, and specifies Runtime Fabric coverage as well as areas that require additional configuration.
```

Comment:
and specifies coverage provided by default as well as how additional methods that can be applied with Runtime Fabric.

Reply:
This got reworded.